tests/common.go

```diff
@@ -552,6 +552,74 @@ func ValidateContext(ctx *scheduler.Context, errChan ...*chan error) {
 	})
 }
 
+func ValidatePureCloudDriveTopologies(errChan ...*chan error) {
```

Comment:
Let's have a return for functions so we can then later on decide how we want to treat it in the test